docs/types.rst

```diff
@@ -813,9 +813,14 @@ for each ``_KeyType``, recursively.
             balances[msg.sender] = newBalance;
         }
     }
+  
+::
+
+    pragma solidity ^0.4.0;
 
     contract MappingUser {
-        address contractAddress = 0x42;
+        // the address of a MappingExample contract
+        address contractAddress = 0x42...;
```

Comment:
I actually meant that it must be a valid literal, but looking at it again we have two options:
- leave as it is
- have it as two contracts means the interface must be provided in the second

```
interface MappingExample {
  function balances(address) returns (uint);
}
```

Thought that might need links to the appropriate sections of the documentation (interfaces and how the public modifier on mappings works).